dev-utils/build.js

```diff
@@ -157,7 +157,7 @@ function getCommonWebpackConfig(bundleType, packageType) {
   const mode = isEnvProduction ? 'production' : 'development'
 
   return {
-    devtool: isEnvProduction ? 'source-map' : false,
+    devtool: isEnvProduction ? false : 'inline-source-map',
```

Comment:
did you try 'inline-cheap-module-source-map' ? inline-source-map is the slowest of all. Also this change affects the production bundles. I hope thats not what you intended.